reanalysis/vep_jobs.py

```diff
@@ -288,65 +282,32 @@ def vep_one(
 
     j = b.new_job('VEP', (job_attrs or {}) | {'tool': 'vep'})
     j.image(image_path('vep'))
-    STANDARD.set_resources(j, storage_gb=50, mem_gb=50, ncpu=16)
+
+    # huge resource reduction
+    STANDARD.set_resources(j, storage_gb=50, mem_gb=16, ncpu=4)
 
     if not isinstance(vcf, hb.ResourceFile):
         vcf = b.read_input(str(vcf))
 
-    if out_format == 'vcf':
```

Comment:
Should these changes be in this PR?

Reply:
ah, this is a bit of bleed from poking around in these scripts. I'll reinstate this, and make all the VEP stuff a separate change